src/AirshipBindings.NETStandard/AirshipBindings.NETStandard.iOS/Airship.cs

```diff
@@ -14,13 +14,26 @@ namespace UrbanAirship.NETStandard
 
     public class Airship : UADeepLinkDelegate, IAirship
     {
-        private static Airship sharedAirship = new Airship();
+        private static Lazy<Airship> sharedAirship = new Lazy<Airship>(() =>
+        {
+            Airship instance = new Airship();
+            instance.Init();
+            return instance;
+        });
+
+        private void Init()
+        {
+            NSNotificationCenter.DefaultCenter.AddObserver(aName: (NSString)"com.urbanairship.channel.channel_created", (channelId) =>
+            {
+                Console.WriteLine(channelId);
```

Comment:
need to call the handler here. What about channel updates? `com.urbanairship.channel.channel_updated`

Reply:
You are still only logging the channel id instead of notifying the listeners.

```
            OnChannelCreation?.Invoke(this, new ChannelEventArgs(channelId));
```

But I am not actually what the event is structured, so you maya need to parse the event from the observer:

something like this:
```
(event) => { OnChannelCreation?.Invoke(this, new ChannelEventArgs(event["com.urbanairship.channel.identifier"]))
```

Reply:
I'm a pretty lost:
Are you telling me to do this on Android or iOS, because this line is one Android side:
`OnChannelCreation?.Invoke(this, new ChannelEventArgs(channelId));`

Reply:
So you are taking a native event and pushing through as a c# event. On Android you add an event listener to Android SDK, then notify C# event listeners with  `OnChannelCreation?.Invoke(this, new ChannelEventArgs(channelId));`

On IOS, you are adding the event listener to the iOS SDK, but instead of notifying the C# event listeners you are just logging the event. You need to trigger the event handlers with `OnChannelCreation?.Invoke(this, new ChannelEventArgs(channelId))` like you do on Android.

Reply:
The argument to the NSNotificationCenter callback Action is going to be an NSNotification. So it should look something like this:

```
NSNotificationCenter.DefaultCenter.AddObserver(aName: (NSString)"com.urbanairship.channel.channel_created", (NSNotification notification) =>
{
    string channelID = notification.UserInfo["com.urbanairship.channel.identifier"];
    OnChannelCreation?.Invoke(this, new ChannelEventArgs(channelId));
}
```

Reply:
Or you can just get the channel from UAirship instead of parsing it out of the notification